### PR TITLE
LPS-195674 Display date object fields without time

### PR DESCRIPTION
**Pull request by guilhermedcamacho**

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:object

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java [hunk 1]
```diff
@@ -138,6 +149,11 @@ public static void setUpClass() throws Exception {
 
 		user1 = TestPropsValues.getUser();
 
+		dtoConverterContext = new DefaultDTOConverterContext(
+			false, Collections.emptyMap(),
+			BaseNotificationTypeTest._dtoConverterRegistry, null,
+			LocaleUtil.getDefault(), null, BaseNotificationTypeTest.user1);
```
Comment (MarinhoFeliphe) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java:155]: Hey @guilhermedcamacho, can we replace `BaseNotificationTypeTest.user1` with `user1`?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:forward

**Comment by guilhermedcamacho** [comment]
ci:stop

**Comment by guilhermedcamacho** [comment]
Manually forwarded to [brianchandotcom#144227](https://github.com/brianchandotcom/liferay-portal/pull/144227)

**Comment by guilhermedcamacho** [comment]
ci:help

**Pull request closed without merging**